CONTRIBUTING.md

```diff
@@ -430,6 +450,14 @@ See [Troubleshooting section](#troubleshooting) for steps you can make to clean
 Once you performed the first build, the images are rebuilt locally rather than pulled - unless you
 force pull the images. But you can force it using the scripts described below.
 
+## Automated image building
+
+You can also export one of the three variables to control the behaviour of image rebuilding:
+
+* Images will be automatically rebuild when needed: `ASSUME_YES="true"`
```

Comment:
`ASSUME_YES` is really vague and doesn't tell me what I'm assuming yes to. Could you please change this to something like "ASSUME_AIRFLOW_IMAGE_REBUILD"?

Reply:
It's a bit general because it will also confirm other stuff like removing docker images (it's a genera Assume yes to all questions :). But I can rename it slightly indeed.

Reply:
It's modelled after some linux switches (-y = assume yes to all questions )

Reply:
For example the same is used when we remove images .

Reply:
I added more general and comprehensive description.